packages/mobile/src/redux/store.ts

```diff
@@ -19,10 +19,10 @@ let lastEventTime = Date.now()
 
 const persistConfig: PersistConfig<RootState> = {
   key: 'root',
-  version: 34, // default is -1, increment as we make migrations
+  version: 35, // default is -1, increment as we make migrations
   keyPrefix: `reduxStore-`, // the redux-persist default is `persist:` which doesn't work with some file systems.
   storage: FSStorage(),
-  blacklist: ['geth', 'networkInfo', 'alert', 'imports'],
+  blacklist: ['geth', 'networkInfo', 'alert', 'imports', 'supercharge'],
```

Comment:
What is this blacklist for?

Reply:
It's so that the state is not stored in the local storage between sessions. Like if you kill the app and re-open it the `supercharge` state is reset to its `initialState`. Since there's nothing there worth saving I added it to the blacklist